formwidgets/menuitems/assets/js/menu-items-editor.js

```diff
@@ -188,6 +188,10 @@
                 $.each(val, function(vbProperty, vbVal) {
                     var $input = $('[name="viewBag['+vbProperty+']"]', $popupContainer).not('[type=hidden]')
                     setPropertyOnElement($input, vbVal)
+                    if (vbProperty.startsWith('localeTitle')) {
+                        var locale = $input.data('locale');
+                        $('[name="RLTranslate['+locale+'][title]"]', $popupContainer).val(vbVal);
```

Comment:
@mjauvin Standard JS code practice with October is to not use semi-colons.
```suggestion
                        var locale = $input.data('locale')
                        $('[name="RLTranslate['+locale+'][title]"]', $popupContainer).val(vbVal)
```